Modules/IO/MINC/include/itkMINCImageIO.h

```diff
@@ -153,6 +158,7 @@ class ITKIOMINC_EXPORT MINCImageIO : public ImageIOBase
 
   // complex type images, composed of complex numbers
   // int m_Complex;
+  bool m_RAS_to_LPS;
```

Comment:
Please, do not use underscores in variable names: ITK uses medial case.

Reply:
Sorry I haven't heard of medial case. What's that?

Reply:
Camel case.